osutil/mockable.go

```diff
@@ -0,0 +1,84 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package osutil
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/user"
+	"syscall"
+)
+
+const (
+	// ProcSelfMountInfo is a path to the mountinfo table of the current process.
+	ProcSelfMountInfo = "/proc/self/mountinfo"
+)
+
+var (
+	userLookup  = user.Lookup
```

Comment:
Most of these are not mockable either, are they? We might need to add a comment that some things here are mockable only internally (via export_test.go) and some things are mockable from the outside.